client/tests/test_backend.py

```diff
@@ -35,9 +35,12 @@ def test_save_get_list_local_storage(self):
         with self.assertRaises(ValueError):
             self.local_storage.get(trial_id="999")
         # List
-        list_trials = self.local_storage.list()
+        list_trials = self.local_storage.list(query="keep_trial")
         self.assertTrue(isinstance(list_trials, list))
         self.assertTrue(isinstance(list_trials[0], Trial))
+        list_trials = self.local_storage.list(query="trial999")
```

Comment:
We should test that we get a response that nothing was found by a message or raise an error. Up to you to choose :)